From 4fec034632dbc06c138cf0ec9ec0d539288e86f8 Mon Sep 17 00:00:00 2001
From: Martin Jansa <martin.jansa@gmail.com>
Date: Mon, 28 Sep 2026 22:56:35 +0200
Subject: [PATCH] PlayerFactory.cpp: fix build with newer libc++

Fixes:
g-media-pipeline-clang/1.0.0-gav.59/sources/g-media-pipeline-clang-1.0.0-gav.59/src/playerfactory/PlayerFactory.cpp:82:3: error: use of undeclared identifier 'transform'
|    82 |   transform(uri.begin(), prot_i,std::back_inserter(protocolType),
|       |   ^~~~~~~~~

Upstream-Status: Pending [webOS OSE is no longer maintained on github]
Signed-off-by: Martin Jansa <martin.jansa@gmail.com>
---
 src/playerfactory/PlayerFactory.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/playerfactory/PlayerFactory.cpp b/src/playerfactory/PlayerFactory.cpp
index 5bb4aad..96b238c 100644
--- a/src/playerfactory/PlayerFactory.cpp
+++ b/src/playerfactory/PlayerFactory.cpp
@@ -19,6 +19,7 @@
 #include <glib-unix.h>
 #include <gst/gst.h>
 #include <pbnjson.hpp>
+#include <algorithm>
 #include <regex>
 
 #include "PlayerFactory.h"
